.github/workflows/ci.yml

```
@@ -93,7 +93,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

```suggestion
          node-version: 18.17
```
18.18 apparently has problems with embroider